pkg/static/login.js

```diff
@@ -345,10 +345,6 @@
             el.focus();
         });
 
-        const os_release = environment["os-release"];
```

Comment:
Removing this is the right idea, but it _might_ break external packages, no?

Reply:
Theoretically yes, but it was never documented and is hard to just find by accident. I checked all of our own projects, session-recording, subscription-manager, and all the other 3rd party apps on https://cockpit-project.org/applications.html , and nothing is using that.

Reply:
Would you like me to drop this commit, or was this just a question how thoroughly I checked?